### PR TITLE
increase test coverage

### PR DESCRIPTION
**Pull request by millken**

### CODE REVIEW

File: testutil/file_test.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFile(t *testing.T) {
+	assert := assert.New(t)
+	tmpPath, err := PathOfTempFile("iotx")
```
Comment (Liuhaai) [inline testutil/file_test.go:17]: Could we use `os.Stat()` to check its existence after it is created, and check again after do cleanup

File: gasstation/gasstattion_test.go [hunk 2]
```diff
@@ -201,6 +201,36 @@ func TestEstimateGasForAction(t *testing.T) {
...
+	require.False(gs.IsSystemAction(sel))
+
+	gb := action.GrantRewardBuilder{}
+	grant := gb.Build()
+	act = builder.SetAction(&grant).Build()
+	sel, err = action.Sign(act, identityset.PrivateKey(1))
+	require.NoError(err)
+	require.True(gs.IsSystemAction(sel))
+
+	act2 := action.NewPutPollResult(1, 1, nil)
```
Comment (Liuhaai) [inline gasstation/gasstattion_test.go:229]: act2 -> actPollResult

File: gasstation/gasstattion_test.go [hunk 3]
```diff
@@ -201,6 +201,36 @@ func TestEstimateGasForAction(t *testing.T) {
...
+	builder := action.EnvelopeBuilder{}
+	cf := action.ClaimFromRewardingFundBuilder{}
+	cfb := cf.Build()
+	act := builder.SetAction(&cfb).Build()
+	sel, err := action.Sign(act, identityset.PrivateKey(1))
+	require.NoError(err)
+	require.False(gs.IsSystemAction(sel))
+
+	gb := action.GrantRewardBuilder{}
+	grant := gb.Build()
```
Comment (Liuhaai) [inline gasstation/gasstattion_test.go:223]: grant -> actGrantReward

File: gasstation/gasstattion_test.go [hunk 4]
```diff
@@ -201,6 +201,36 @@ func TestEstimateGasForAction(t *testing.T) {
...
+	require.Equal(ret, uint64(0))
+}
+
+func TestIsSystemAction(t *testing.T) {
+	require := require.New(t)
+	gs := NewGasStation(nil, nil, nil, config.Default.API)
+	require.NotNil(gs)
+	builder := action.EnvelopeBuilder{}
+	cf := action.ClaimFromRewardingFundBuilder{}
+	cfb := cf.Build()
```
Comment (Liuhaai) [inline gasstation/gasstattion_test.go:216]: cfb -> actClaimFromRewarding

File: action/action_test.go [hunk 5]
```diff
@@ -79,3 +80,39 @@ func TestActionProtoAndVerify(t *testing.T) {
...
 	})
 }
+
+func TestActionClassifyActions(t *testing.T) {
+	require := require.New(t)
+	producerAddr := identityset.Address(27).String()
+	producerPriKey := identityset.PrivateKey(27)
+	amount := big.NewInt(0)
+	selp0, err := SignedTransfer(producerAddr, producerPriKey, 1, amount, nil, 100, big.NewInt(0))
+	require.NoError(err)
```
Comment (Liuhaai) [inline action/action_test.go:90]: b/c we are to test `ClassifyActions()` here , I think no need to check SignedTransfer()'s err.
```
func TestActionClassifyActions(t *testing.T) {
	require := require.New(t)
	var (
		producerAddr   = identityset.Address(27).String()
		producerPriKey = identityset.PrivateKey(27)
		amount         = big.NewInt(0)
		selp0, _       = SignedTransfer(producerAddr, producerPriKey, 1, amount, nil, 100, big.NewInt(0))
		selp1, _       = SignedTransfer(identityset.Address(28).String(), producerPriKey, 1, amount, nil, 100, big.NewInt(0))
		selp2, _       = SignedTransfer(identityset.Address(29).String(), producerPriKey, 1, amount, nil, 100, big.NewInt(0))
		selp3, _       = SignedExecution(producerAddr, producerPriKey, uint64(1), amount, uint64(100000), big.NewInt(10), []byte{})
		selp4, _       = SignedExecution(producerAddr, producerPriKey, uint64(2), amount, uint64(100000), big.NewInt(10), []byte{})
	)
	actions := []SealedEnvelope{selp0, selp1, selp2, selp3, selp4}
	tsfs, exes := ClassifyActions(actions)
	require.Equal(len(tsfs), 3)
	require.Equal(len(exes), 2)
}
```

**Review by Liuhaai — APPROVED** [review]

**Review by raullenchai — APPROVED** [review]

**Pull request merged**